@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -885,39 +884,10 @@ watch(
       }
     }
 
-    .install-custom-config {
-      display: flex;
-      margin-block-start: 3rem;
-      gap: 4.2rem;
-    }
-
-    .install-ssh-key-container {
-      margin-block-start: 3rem;
-    }
-
-    .install-ssh-key {
-      display: flex;
-      align-items: center;
-      gap: 0.5rem;
-      width: 50%;
-    }
-
-    .install-chips {
-      display: flex;
-      gap: 0.5rem;
-      margin-block-end: 1rem;
-    }
-
```

Comment:
I think you can also delete the commented code related to these classes in the template